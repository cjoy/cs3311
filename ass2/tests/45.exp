Timothy Cappie-Wood (3363921)
Suggestions for study in 10s2

Course   Title                 UOC Requirement
-------- -----                ---- -----------
COMP1927 Computing 2             6 Level 1 core
MATH1231 Mathematics 1B          6 Level 1 core
MATH1241 Higher Mathematics 1    6 Level 1 core
COMP2041 Software Constructio    6 Level 2 core
Free.... Free Electives (many   42 Free elect
