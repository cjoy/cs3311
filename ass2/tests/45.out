Timothy Cappie-Wood (3363921)
Suggestions for study in 10s2

164 165Course   Title                 UOC Requirement
-------- -----                ---- -----------
